spec/requests/resident_registers_visitor_request_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe 'Resident registers visitor request' do
```

Comment:
Não existe um teste de sucesso que verifica se o Job está enfileirado? Poderia ter um teste de falha também que garante que o Job não é enfileirado se algo der errado.

Reply:
No arquivo dos testes do job temos essa linha `expect(UpdateVisitDateJob).to have_been_enqueued.with(first_visitor).at((Date.current + 1.day).to_datetime)` que verifica a existencia do job na fila

Reply:
Ja o teste de falha vamos implementar